### PR TITLE
mgr/dashboard: Add ui-api endpoints to the API docs

### PR DESCRIPTION
**Pull request by aaSharma14**
This PR intends to add ui-api endpoints to swagger ui

Fixes:https://tracker.ceph.com/issues/53318
Signed-off-by: Aashish Sharma <aasharma@redhat.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by nizamial09** [comment]
jenkins test dashboard

**Comment by aaSharma14** [comment]
jenkins test dashboard

**Comment by aaSharma14** [comment]
jenkins test make check

**Comment by aaSharma14** [comment]
jenkins test dashboard cephadm

**Comment by aaSharma14** [comment]
jenkins test dashboard

**Comment by aaSharma14** [comment]
jenkins test dashboard

**Comment by aaSharma14** [comment]
jenkins test dashboard

**Comment by aaSharma14** [comment]
jenkins test dashboard

**Comment by aaSharma14** [comment]
jenkins test dashboard

**Comment by aaSharma14** [comment]
jenkins test dashboard cephadm

File: src/pybind/mgr/dashboard/frontend/src/app/core/navigation/api-docs/api-docs.component.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ import SwaggerUI from 'swagger-ui';
 export class ApiDocsComponent implements OnInit {
   ngOnInit(): void {
     SwaggerUI({
-      url: window.location.origin + '/docs/openapi.json',
+      url: window.location.origin + '/docs/api-all.json',
```
[the remark this answers is not in the document]
Comment (avanthakkar) [reply src/pybind/mgr/dashboard/frontend/src/app/core/navigation/api-docs/api-docs.component.ts:13]: OK! So why we having 2 different types now api & ui-api ?

**Comment by avanthakkar** [comment]
jenkins test dashboard cephadm

**Comment by avanthakkar** [comment]
jenkins test dashboard

**Pull request closed without merging**